### DISCUSSION TITLE
Hide global navigation as user

### DISCUSSION THREAD

**Discussion by MaxWinsemius** [requarks/wiki#5322] [General]
Is there a way to hide the global navigation menu? It does not auto close on higher resolutions than 720 pixels wide

**Comment by jamsondu** [comment]
Touch the screen left.

Only touch screen is supported.

**Reply by MaxWinsemius** [reply]
Are there any plans for desktop support?

**Reply by jamsondu** [reply]
Global: 
disable site navigation
[https://docs.requarks.io/en/navigation](https://docs.requarks.io/en/navigation)

**By Page:**
Edit Page: Copy the code to "Page Properties -> Style"
`
.v-navigation-drawer {
  visibility: hidden
}
.v-main {
  padding-left: 0!important
}
`

**Reply by MaxWinsemius** [reply]
This seems pretty much as a server-side implementation, instead of a client-side implementation, right?

**Reply by jamsondu** [reply]
Yes, hopefully the client-side implementation will be supported in later versions.
